DB/api/test_query_db.py

```diff
@@ -84,6 +84,8 @@ def test_classify(self):
         self.retrieve_args[0] = 5
```

Comment:
Please update this test to reflect the fact that you replaced the Analysis_3 library.